runtime/queries/bash/highlights.scm

```diff
@@ -1,84 +1,88 @@
 [
- "(" ")"
- "{" "}"
- "[" "]"
- "[[" "]]"
- "((" "))"
+  "("
```

Comment:
Matching brackets should remain grouped (in other languages as well).

Reply:
Disagree. This is not automatable.

Reply:
It doesn't have to be when `; format-ignore` exists.

Reply:
Yes, but these defeat the purpose. We should use them _very_ sparingly, if autoformatting produces inacceptable results.